<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveSetting</class>
 <widget class="QWidget" name="SaveSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>56</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>56</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="save_pic_btn">
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/uiframe/images/config/no_selected.svg</normaloff>
          <normalon>:/uiframe/images/config/selected.svg</normalon>:/uiframe/images/config/no_selected.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="save_pic_title">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>359</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="save_pic_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="save_type_and_path_title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Saving Type and Path</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item row="1" column="1">
         <widget class="QLabel" name="save_ng_title">
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>NG Image</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="ng_path_btn">
          <property name="minimumSize">
           <size>
            <width>76</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="save_format_title">
          <property name="text">
           <string>Image Format</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="Line" name="line_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 0px;
background: #eeeeee;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="ok_path">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="ok_path_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>76</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>32</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="save_ok_title">
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>OK Image</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="ng_path">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QComboBox" name="save_format_list">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>bmp</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>jpg</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>png</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="save_ng_btn">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/uiframe/images/config/no_selected.svg</normaloff>
            <normalon>:/uiframe/images/config/selected.svg</normalon>:/uiframe/images/config/no_selected.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="QWidget" name="widget_4" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="save_ok_btn">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/uiframe/images/config/no_selected.svg</normaloff>
            <normalon>:/uiframe/images/config/selected.svg</normalon>:/uiframe/images/config/no_selected.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>14</width>
            <height>14</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QWidget" name="widget_5" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>1</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px;
background: #eeeeee;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="delete_condition_btn">
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/uiframe/images/config/no_selected.svg</normaloff>
             <normalon>:/uiframe/images/config/selected.svg</normalon>:/uiframe/images/config/no_selected.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="delete_condition_title">
           <property name="text">
            <string>Delete Setting</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>229</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="save_day_title">
          <property name="text">
           <string>The saving days over</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLabel" name="save_number_warning">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="save_number_title">
          <property name="text">
           <string>The number of pictures over</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLabel" name="save_day_warning">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="vtk::display::LineEdit" name="save_number_input">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="vtk::display::LineEdit" name="save_day_input">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vtk::display::LineEdit</class>
   <extends>QLineEdit</extends>
   <header>uiframe/commonuiset/lineedit/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
